stores/amazon.py

```diff
@@ -121,8 +123,10 @@ def login(self):
             log.info("Email not needed.")
             pass
 
+        log.info("Remember me checkbox")
+        selenium_utils.button_click_using_xpath(self.driver, '//*[@name="rememberMe"]')
+
         log.info("Password")
```

Comment:
As above - is this log useful to end users?

Reply:
It's mainly useful for when users have issues on that page and we can figure out where it went wrong without getting their debug logs.